ClientApp/viking-vault/src/UserDashboard/ProfilePage.css

```diff
@@ -1,13 +1,13 @@
 .ProfilePage 
```

Comment:
Keep the CSS naming convention consistent

Reply:
Come here and ally with me against the person that did this atrocity, in the very first week